docs/concept-exercises.md

```diff
@@ -43,7 +43,7 @@ When designing Concept Exercises, please consider the following guidelines:
 
 1. The exercise should be able to be solved in 5-10 minutes by someone proficient in the language.
 1. The exercise should not involve having to learn or engineer a non-trivial algorithm.
```

Comment:
I'm still unsure what a "trivial algorithm" is. Should that go in another PR or can it be discussed here?

Reply:
We can discuss it here.

Reply:
@SaschaMann I'll merge this and then we can submit a new PR later.